RegisterController.js

```diff
@@ -6,6 +6,12 @@
 import {AccountService} from './AccountService.js';
 import pDebounce from 'p-debounce';
 
+/**
+ * Encapsulates the registration methods.
+ * @param {Object} options
```

Comment:
I guess `options` itself is optional here.

Reply:
yeah true. I'll change the other ones too.

Reply:
btw the constructor for AccountService assigns what we now call options to a config object.
```
    this.config = {urls};
```